recipes/jetty.rb

```diff
@@ -93,17 +93,29 @@
   end
 end
 
+solr_config_path =  case node['platform_family']
```

Comment:
i think it would be good to move this to default attributes, like:

```ruby
default['solrcloud']['sysconfig_file'] = value_for_platform_family(
  'debian' => '/etc/default/solr',
  'rhel' => '/etc/sysconfig/solr'
)
```